src/main/kotlin/no/nav/syfo/client/PdfgenClient.kt

```diff
@@ -19,10 +23,16 @@ class PdfgenClient constructor(
     private val httpClient: HttpClient
 ) {
     suspend fun createPdf(payload: PdfPayload): ByteArray = retry("pdfgen") {
-        httpClient.get<ByteArray>(url) {
+        val httpResponse = httpClient.get<HttpStatement>(url) {
             contentType(ContentType.Application.Json)
             method = HttpMethod.Post
             body = payload
+        }.execute()
+        if (httpResponse.status == HttpStatusCode.OK) {
```

Comment:
Grunnen til at jeg gjør dette er for å unngå at en evt feil-respons skal tolkes som en respons, og så går vi bare videre. Siden en bytearray er såpass generelt tror jeg ikke det ville ha feilet. Men mulig jeg tenker feil nå :)